### PR TITLE
Added group availability scheduler + meeting time

### PR DESCRIPTION
**Pull request by mattgerges**
- Added group availability generator so that the overlap in availability can be displayed on the groups page

- Added group meeting time string generator so that the meeting time can be displayed on the groups page

### CODE REVIEW

**Comment by mattgerges** [comment]
Completed handlers + routes

File: cs130_back/handlers/group.go [hunk 1]
```diff
@@ -62,18 +63,104 @@ func CreateGroup(db *gorm.DB, w http.ResponseWriter, r *http.Request) {
...
+// GetGroup retrieves and returns the group
+func GetGroup(db *gorm.DB, w http.ResponseWriter, r *http.Request) {
+	vars := r.URL.Query()
+	id, ok := strconv.Atoi(vars["g_id"][0])
+	if ok != nil {
+		respondWithError(w, http.StatusBadRequest, "Invalid group id")
+		return
+	}
+	group := models.Group{ID: id}
+	if GetGroupByID(db, &group, w) == 0 {
```
Comment (shirlyfang) [inline cs130_back/handlers/group.go:97]: For a group, you need to get their actual members` User objects as well. You also need to get the meeting time for this group.

**Review by shirlyfang — CHANGES_REQUESTED** [review]
Seems like this would complete issue #37. Left a comment on whats needed in the response.

**Review by shirlyfang — APPROVED** [review]
lgtm for issue #37.

**Review by Acabado — APPROVED** [review]

**Pull request merged**